wordpress_rock/rockcraft.yaml

```diff
@@ -0,0 +1,188 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: wordpress
+summary: Wordpress rock
+description: Wordpress OCI image for the Wordpress charm
+base: ubuntu:20.04
+# base: bare
+# build-base: ubuntu:20.04
+run-user: _daemon_
+license: Apache-2.0
+version: "1.0"
+platforms:
+  amd64:
+parts:
+  apache2:
+    plugin: dump
+    source: files
+    build-packages:
+    - apache2
+    - bzr
+    - curl
+    - git
+    - libapache2-mod-php
+    - libgmp-dev
+    - php
+    - php-curl
+    - php-gd
+    - php-gmp
+    - php-mysql
+    - php-symfony-yaml
+    - php-xml
+    - pwgen
+    - python3
+    - python3-yaml
+    - unzip
+    overlay-packages:
+    - apache2
+    - bzr
```

Comment:
If I'm not mistaken, bzr, git and curl are only used to install the wp plugins, so they might better be placed among the `build-packages` of the `wordpress` part

Reply:
I moved `bzr`, `git`, `curl` and `unzip` to `build-packages` for the `wordpress` part. Initially I was thinking about keeping them in the final image as we can use them when ssh-ing into the running container.

Reply:
It seems that removing those packages from the final image will cause problems here : https://github.com/canonical/wordpress-k8s-operator/blob/55a3f6ac915b5bd40872fa6e64992b5102944f7a/tests/integration/helper.py#L472. I'll investigate further but for now I'm adding them back to `overlay-packages` and I'll run some local tests to see what dependencies are required when using those packages.

Reply:
got it, the problem comes from `ca-certificates` which is a dependency of `git` and `bzr`. I'll add this to `overlay-packages` and move back the other packages to their appropriate location.